src/contract_wrappers/tokens/security_token_wrapper/common.ts

```diff
@@ -47,36 +13,31 @@ import {
   RestrictedPartialSaleTMContract_3_1_0,
   TxData,
   Web3Wrapper,
-  LogWithDecodedArgs,
   BigNumber,
   ethersUtils,
   CappedSTOContract_3_1_0,
   USDTieredSTOContract_3_1_0,
 } from '@polymathnetwork/abi-wrappers';
-import { schemas } from '@0x/json-schemas';
-import assert from '../../utils/assert';
-import ERC20TokenWrapper from './erc20_wrapper';
-import ContractFactory from '../../factories/contractFactory';
+import assert from '../../../utils/assert';
+import ERC20TokenWrapper from '../erc20_wrapper';
+import ContractFactory from '../../../factories/contractFactory';
 import {
-  EventCallback,
   Feature,
   FULL_DECIMALS,
   FundRaiseType,
-  GetLogs,
-  GetLogsAsyncParams,
   ModuleName,
   ModuleType,
   Partition,
   PERCENTAGE_DECIMALS,
   Perm,
-  Subscribe,
-  SubscribeAsyncParams,
   TxParams,
   CappedSTOFundRaiseType,
   TransferStatusCode,
   ErrorCode,
   ContractVersion,
-} from '../../types';
+  Subscribe,
```

Comment:
Isn't this import unused?

Reply:
nop, I had to defined:

```
public subscribeAsync!: Subscribe;
public getLogsAsync!: GetLogs;
```

otherwise the project did not compiled